src/redux/features/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: () => ['Under construction'],
+  },
+});
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225294611-fc9b0a68-3425-428d-9cc6-facf11f2399c.png)

- [ ] Since you don't wanna have any questions in FAQ section then it would be nice if you remove your FAQ section to make your readme file descriptive and professional :blush:

Reply:
Thanks 
@HAMMAS-SALEEM . Done! Pls can you check it again??!